komorebic/src/main.rs

```diff
@@ -1169,7 +1172,12 @@ fn main() -> Result<()> {
             let mut current_exe = std::env::current_exe().expect("unable to get exec path");
             current_exe.pop();
 
-            let komorebic_exe = current_exe.join("komorebic.exe");
+            let bin = if args.no_console {
```

Comment:
I think it probably makes sense to drop this `no_console` arg and just default to `komorebic-no-console` for autostart since I'm pretty sure everybody hates that window flash 😅

Reply:
cool